src/Uno.UI.RemoteControl.VS/IDEChannel/IDEChannelClient.cs

```diff
@@ -98,6 +99,10 @@ private void ProcessDevServerMessage(object sender, IdeMessageEnvelope devServer
 				case KeepAliveIdeMessage:
 					_logger.Verbose($"Keep alive from Dev Server");
 					break;
+				case NotificationRequestIdeMessage e when e is { } message:
+					_logger.Verbose($"Dev Server will open the Notification Message with message {e.Message}");
+					process = OnMessageReceived.InvokeAsync(this, message);
```

Comment:
The idea to have a generic `OnMessageReceived` was to remove dependency to all type of message from the `IDEChannelClient` (i.e. remove that switch case). But that's fine for now, let merge this PR and address it later.

Reply:
@rafael-rosa-knowcode